packages/shared/src/schemas.ts

```diff
@@ -127,6 +127,7 @@ export const openSwapDepositChannelSchema = z
         intermediateAmount: z.string().optional(),
         egressAmount: z.string(),
         estimatedPrice: z.string(),
+        recommendedSlippageTolerancePercent: z.number().optional(),
```

Comment:
think this can be non-optional

Reply:
I feel like since it's not mandatory for the main purpose of this endpoint, would be better to have it as optional. would make it more error proof.
what do you think?

Reply:
fine for me as well - not a strong opinion 🙂